packages/model-viewer/src/features/scene-graph/material.ts

```diff
@@ -321,7 +321,6 @@ export class Material extends ThreeDOMElement implements MaterialInterface {
     const enableTransparency =
         (material: MeshStandardMaterial, enabled: boolean): void => {
           material.transparent = enabled;
-          material.depthWrite = !enabled;
```

Comment:
Is this change intentional?

Reply:
Yes, there was a long discussion years ago regarding the best way to render transparency and this was the result. @donmccurdy do you remember that? I wonder though how much it has been superseded by the actual glTF transparency extension which came along afterwards.

Reply:
It would be a pretty major regression to enable depth write for all alpha blended materials I think.   There's probably a way to solve this in post-processing, like a depth pre-pass... perhaps we could leave that for a separate PR?

Reply:
I don't think something like that exists in postprocessing, if not then this is definitely , but I guess its possible to do a depth pre-pass to writes all materials as depthWrite = true, saving that to the texture which is used by the depth-requiring effects.